earthpy/spatial.py

```diff
@@ -40,7 +40,7 @@ def extent_to_json(ext_obj):
 
 # calculate normalized difference between two arrays
 # both arrays must be of the same size
-def normalized_diff(b1, b2):
+def normalized_diff(b2, b1):
```

Comment:
@joemcglinchy can you kindly remove this fix in this PR so it's distinct from the tif generator? I'd like to hold off on implementing this for my students sake! but i want to implement it. just not in this PR.